### DISCUSSION TITLE
PROXY_HTTP_BIND disables ManagementCA

### DISCUSSION THREAD

**Discussion by vacan1t** [Keyfactor/ejbca-ce#420] [Q&A]
Hey! I am trying out EJBCA CE on Docker and trying to out some production like config with `TLS_SETUP_ENABLED=true` and a NGINX proxy in front for TLS server termination and client cert passthrough. 

According to my understanding of the docs [https://hub.docker.com/r/keyfactor/ejbca-ce](https://hub.docker.com/r/keyfactor/ejbca-ce) we should set `PROXY_HTTP_BIND=x.x.x.x` when using a proxy, but that seems to disable the automatic creation of ManagemenCA and enrollment of superadmin, even though the docs clearly states that `TLS_SETUP_ENABLED=true` should do this for us.

Why is that? And what is actually recommended settings when using a proxy but also want the initial ManagementCA?

**Comment by primetomas** [comment]
See for example: #614 and #604 

We have some new container documentation, including Helm charts to deploy EJBCA to kubenteres, with optional ingress.
Take a look: https://docs.keyfactor.com/container/latest/ejbca/deployment

**Reply by MaximShepelev** [reply]
The behaviour is still such as described in this issue and not mentioned in documentation

**Reply by primetomas** [reply]
I updated the documentation on DockerHub with a note about this. Thanks for the suggested update.
